packages/file-type-icons/src/getFileTypeIconAsHTMLString.ts

```diff
@@ -0,0 +1,28 @@
+import { DEFAULT_BASE_URL } from './initializeFileTypeIcons';
+import {
+  getFileTypeIconNameFromExtensionOrType,
+  getFileTypeIconSuffix,
+  DEFAULT_ICON_SIZE,
+  IFileTypeIconOptions,
+} from './getFileTypeIconProps';
+
+export function getFileTypeIconAsHTMLString(
```

Comment:
Can you please add a comment to this? Also the function is not being exported through the index file. And you will probably need to run `yarn update-api` to update the api-extractor definition of the library.

Reply:
Added a comment and exported through the index file. I couldn't find an existing api.md or api-extractor.json file for this package. Should I add one?

Reply:
@dzearing gentle ping! :)

Reply:
We should add an API extractor config and API file for this package (same pattern as other packages).